src/fractal_feature_explorer/config.py

```diff
@@ -22,17 +22,23 @@ def remove_trailing_slash(value: str) -> str:
     return value.rstrip("/")
 
 
-class LocalConfig(BaseModel):
+class BaseConfig(BaseModel):
     model_config = ConfigDict(extra="forbid")
-    deployment_type: Literal["local"]
+    deployment_type: Literal["local", "production"]
+    allow_local_paths: bool
+    cache_ttl: float | timedelta | str | None = None
```

Comment:
Minor comment:

What is the goal of this flexibility?
If `cache_ttl` is defined as part of a configuration file, I guess it would always be a string or float? Or is there another way to set it directly to a `timedelta`?

(also: can we add an example of a valid config somewhere? e.g. would it look like `cache_ttl="60s"` on-disk? or would it be `cache_ttl=60`?)